Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/MerchantPushNotification.yaml

```diff
@@ -0,0 +1,34 @@
+imports:
+  MerchantOperatingCity: Domain.Types.Merchant.MerchantOperatingCity
+  Merchant: Domain.Types.Merchant
+  Language: Kernel.External.Types
+  Text: Data.Text
+
+MerchantPushNotification:
+  tableName: merchant_push_notification
+
+  derives: "Generic,Show,ToJSON,FromJSON"
+
+  types:
+    NotificationKey:
+      enum: "FIRST_NOTIFICATION_RIDE_START,SECOND_NOTIFICATION_RIDE_START,RIDE_ALLOCATED,RIDE_ASSIGNED"
```

Comment:
No need to make Enum for this key. Lets make it text type.

Reply:
done